homeassistant/components/sensor/buienradar.py

```diff
@@ -38,6 +40,10 @@
 # Key: ['label', unit, icon]
 SENSOR_TYPES = {
```

Comment:
I think that we should just kill this sensor and have people use the weather platform instead. If people want to get that data they could always decide to use a template sensor to extract it.